docs/high-performance-order-storage/guide-large-store.md

```diff
@@ -0,0 +1,134 @@
+# A large store's guide to enable HPOS on WooCommerce
+
+Do you run a high-volume WooCommerce store? Are you about to embark on the task to enable HPOS on said store? As we move more and more high-volume store to HPOS, we decided to document the guidelines for those of you who may be faced with this task.
+
+## Before you begin: Get familiar with the HPOS project and related documentation
```

Comment:
Might be a good opportunity to link out to the HPOS cat page in the docs site: https://developer.woo.com/docs/category/hpos/